avaframe/data/avaDamBreak/Inputs/damBreak_com1DFACfg.ini

```diff
@@ -103,7 +103,7 @@ entDefResistance = 0
 
 #+++++++++++++ Output
 # desired result Parameters
-resType = FD_FV
+resType = FD_FV_pfv
```

Comment:
Is there a reason to mix upper and lower case letters here?

Reply:
the non-peak fields use capital letters so far..